### PR TITLE
I58 reorder navbar

### LINKED ISSUE #58: Reorder navbar
## Basic Information

Current navbar order: Home Contact [Visit] Profile Incident

New navbar order: Home [Visit] Contact

- [x] Remove Profile and Incident
- [x] Reorder navbar

## Other Information

Potential scopes:

### PR DESCRIPTION
**Pull request by NicholasChoong**
## Change Summary
- Reordered navbar
- Removed profile and incidents page

- added default colours in tailwind config
- moved path check for the navbar indicator into a useEffect hook so it wont rerun every render

# Related Issue

- Resolve #58

### CODE REVIEW

File: src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -1,50 +1,39 @@
...
+    const pathStart = asPath.split('/')[1]
+    const currentPageTitle =
+      pathStart === ''
+        ? 'Home'
+        : `${pathStart.charAt(0).toUpperCase()}${pathStart
+            .slice(1)
+            .toLowerCase()}`
+    setCurrentPage(currentPageTitle)
+  }, [asPath])
+
```
Comment (APM246) [inline src/components/NavBar/index.tsx:25]: I don't think this refactor is necessary as setCurrentPage is already called when a navlink is clicked (onMouseDown property). This code causes NavBar to be re-rendered one additional time

**Review by APM246 — CHANGES_REQUESTED** [review]

File: src/components/NavBar/index.tsx (hunk 1 above)
Comment (skyheat) [reply src/components/NavBar/index.tsx]: good catch arun 👍

Comment (NicholasChoong) [reply src/components/NavBar/index.tsx]: Thanks for testing this out for me

**Review by skyheat — APPROVED** [review]
Looks ok

**Pull request merged**